tests/reproducibility/test_attri2vec.py

```diff
@@ -45,18 +45,16 @@ def unsup_attri2vec_model(generator, optimizer, bias, normalize):
     return model
 
 
-def unsup_attri2vec(
-    g,
-    optimizer,
-    batch_size=4,
-    epochs=4,
-    bias=True,
-    normalize="l2",
-    number_of_walks=1,
-    walk_length=5,
-    seed=0,
-    shuffle=True,
-):
+def unsup_attri2vec(g, batch_size, shuffle):
+
+    epochs = 4
+    bias = True
+    normalize = "l2"
+    seed = 0
+    optimizer = tf.optimizers.Adam(1e-3)
```

Comment:
Do you think it's useful to specify these things upfront like this? I suspect it's probably ok to push these down to `unsup_attri2vec_model` and `attri2vec_link_pred_model`?